src/ucs/datastruct/piecewise_func.h

```diff
@@ -63,29 +64,29 @@ double ucs_piecewise_func_apply(const ucs_piecewise_func_t *func, size_t x);
 
 
 /**
- * Add the trend on the provided range to the piecewise function.
+ * Add the linear function on the provided range to the piecewise function.
  *
- * @param [inout]  func    Piecewise function to update.
- * @param [in]     start   Start of the range(inclusive).
- * @param [in]     end     End of the range(inclusive).
- * @param [in]     trend   Trend to add.
+ * @param [inout]  func          Piecewise function to update.
```

Comment:
it seems we use "func" name for variables of both linear_func and piecewise_func types.
maybe rename all piecewise_func variables to "pw_func" ?

Reply:
Good idea, done.

Reply:
there are many more places to rename..

Reply:
Changed.